Power_Of_Any_Number/PowerOfAnyGivenNo.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Problem Statement
+ * 	Program to find if a No. is a power of any Given No. in Java
+ *	
+ *Approach
+ *	comparing the log's value with the floor value of log 
+ */
+import java.util.Scanner;
+
+public class PowerOfAnyGivenNo {
+
+	public static void main(String[] args) {
```

Comment:
Please follow 4 space indentation across the file. 
Here, you have used TAB instead.

Reply:
actually i have selected 4 and indent- Space, while committing  the code this time. Do Check Now!